### PR TITLE
Update readme page

### PR DESCRIPTION
**Pull request by hyounes4560**
- Updates the README page with more content/links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,22 +3,34 @@
 [![](https://github.com/buildpacks/docs/workflows/Deploy/badge.svg)](https://github.com/buildpacks/docs/actions)
 [![Gitpod ready-to-code](https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/buildpacks/docs)
 
-### Preview
+## Preview
+
+This repository contains the documentation for Cloud Native Buildpacks (CNBs), pluggable and modular tools that translate source code into OCI images.
+
+Read more about Cloud Native Buildpacks, features, and story behind on our [website](https://buildpacks.io/).
```
Comment (natalieparellano) [inline README.md:10]:
```suggestion
This repository contains the documentation for Cloud Native Buildpacks (CNBs), which are pluggable and modular tools that translate application source code into OCI container images.

Read more about Cloud Native Buildpacks, the features the offer, and the story behind the project on our [website](https://buildpacks.io/).  
```

**Review by natalieparellano — COMMENTED** [review]
Looks good to me! I left one suggestion.

File: README.md (hunk 1 above)
Comment (hyounes4560) [reply README.md]: updated

**Comment by AidanDelaney** [comment]
LGTM, just need the DCO to pass now.

**Comment by AidanDelaney** [comment]
I created a new branch in #729 where the DCO passes.  Will close this branch and merge the other one.

**Pull request closed without merging**